PlugIns/DotScene/README.md

```diff
@@ -42,4 +43,46 @@ To add logic properties to the scene you can use the `<userData>` node as follow
         <property data="1.0" name="mass_radius" type="float" />
     </userData>
 </entity>
-```
\ No newline at end of file
+```
+
+## How to use DotScene
+In recent OGRE3D versions DotScene has been incorporated into the main OGRE repo and made into a Plugin.
+So it has to be loaded in `plugins.cfg`, add the following line:
+```
+Plugin=Plugin_DotScene
+```
+Also, it is required to complie against the library (in a similar fashion to OgreBites) located in: `OgreSDK\ogre-1.12.11\lib\OGRE`
```

Comment:
actually, you do not have to do that.

Reply:
But if I don't do it then there are unresolved symbols at link time...

It was surprinsing because the deprecated method does not need it, unfortunately I am not doing things the way you specified in the manual and that might be the cause.

Reply:
> But if I don't do it then there are unresolved symbols at link time...

yes, you must use the Codec